cmd/keylight/main.go

```diff
@@ -33,6 +33,13 @@ func main() {
 		log.Fatalf("failed to fetch accessory info: %v", err)
 	}
 
+	wifi, err := c.GetWiFiInfo(ctx, d)
```

Comment:
Let's omit this for now, I intend to add more commands to this CLI to dump more verbose information.

Reply:
I removed all the logic for fetching the credentials. I misread and assumed it was available.